pipeline/outputs/dynatrace.md

```diff
@@ -0,0 +1,61 @@
+---
+description: Send logs to Dynatrace
+---
+
+# Dynatrace
+
+Stream logs to [Dynatrace](https://www.dynatrace.com) by utilizing the **http** plugin to send data to [Dynatrace generic log ingest API](https://docs.dynatrace.com/docs/shortlink/lma-generic-log-ingestion).
+
+Before starting you need to get a [Dynatrace API](https://docs.dynatrace.com/docs/shortlink/api-authentication) token with the `logs.ingest` (Ingest Logs) scope, and determine your [environment ID](https://docs.dynatrace.com/docs/shortlink/monitoring-environment#environment-id).  Then configure **http** output plugin with the following HTTP parameters.
```

Comment:
I implemented line wrapping as part of my suggestion and added a "the" for clarity.

Also, I removed the second line, as it was unneeded. It just restated the process to be used in the getting started section. Stating things more than once in the docs just means we have two sources of information that have to be maintained, in the case things change.

```suggestion
Stream logs to [Dynatrace](https://www.dynatrace.com) by utilizing the **http**
plugin to send data to the
[Dynatrace generic log ingest API](https://docs.dynatrace.com/docs/shortlink/lma-generic-log-ingestion).
```